dvc/repo/__init__.py

```diff
@@ -86,29 +89,16 @@ def __init__(self, root_dir=None):
 
         self.updater.check()
 
-    def __repr__(self):
-        return "Repo: '{root_dir}'".format(root_dir=self.root_dir)
-
     @staticmethod
     def find_root(root=None):
-        if root is None:
-            root = os.getcwd()
-        else:
-            root = os.path.abspath(os.path.realpath(root))
-
-        while True:
-            dvc_dir = os.path.join(root, Repo.DVC_DIR)
-            if os.path.isdir(dvc_dir):
-                return root
-            if os.path.ismount(root):
-                break
-            root = os.path.dirname(root)
-        raise NotDvcRepoError(root)
+        return Config.find_root(root=root)
 
     @staticmethod
     def find_dvc_dir(root=None):
-        root_dir = Repo.find_root(root)
-        return os.path.join(root_dir, Repo.DVC_DIR)
+        return Config.find_dvc_dir(root=root)
```

Comment:
same question here

Reply:
it's just so weird to see that Repo is using Config to find its root ... why do we need both methods, btw?